test/com/facebook/buck/jvm/java/testdata/java_dep_ordering/buildables/BUCK.fixture

```diff
@@ -0,0 +1,79 @@
+# Dep graph
```

Comment:
Can we delete these comments about different sorting orders since all we have is lexicographic sorting now

Reply:
Yes, we should, thank you, updated PR.

I kept dep graph though